### PR TITLE
Added test and resources for - Add postBillingAddress method to CartApiWrapper.java

### PR DESCRIPTION
**Pull request by vladbm-work**
This pull request is linked to https://github.com/ralucarobotin/svauto/issues/65

### CODE REVIEW

File: src/main/java/api/domain/cart/items/BillingAddress.java [hunk 1]
```diff
@@ -9,17 +9,17 @@
     private String state;
     private String country;
     @JsonProperty("postal_code")
-    private String postalCode;
+    private String postal_code;
```
Comment (softvision-adrian-rosu) [inline src/main/java/api/domain/cart/items/BillingAddress.java]: this fields should work without underscore, that's why we put the real name in the JsonProperty tag. You just need to generate getters and setters.
It looks like firstName and lastName already works, so postalCode and address1 also should work.

File: src/test/java/api/cart/PostCartTests.java [hunk 2]
```diff
@@ -27,4 +28,13 @@ public void addItemsToCart() {
         Cart postItemToCartResponse = cartApiWrapper.postCartItems(itemToAdd, postCartResponse.getId());
         Assert.assertEquals(cartApiWrapper.statusCode, HttpStatus.SC_OK, "incorrect error code");
     }
+
+    @Test
+    public void addCartBillingAddress() {
+        Cart cart = new Cart();
+        Cart postCartResponse = cartApiWrapper.postEmptycart(cart);
+        BillingAddress billingAddress = CartUtils.addBillingAddressToCart("Vlad", "Baia Mare", "Maramures", "Romania", "437227", "0743188380", "shipping", "Morii", "Nr. 10");
```
Comment (softvision-adrian-rosu) [inline src/test/java/api/cart/PostCartTests.java]: We should avoid using our real address and phone numbers in any public space. For further tests please use a random US address (in order for shipping methods to work later on when we do end-to-end tests).

**Review by softvision-adrian-rosu — CHANGES_REQUESTED** [review]
Please make a rebase with workshop2019 to resolve the conflicts added after Gelu's merge.

File: src/main/java/api/domain/cart/items/BillingAddress.java (hunk 1 above)
Comment (vladbm-work) [reply src/main/java/api/domain/cart/items/BillingAddress.java]: I've set the JsonSetter and JsonGetter but the test keeps failing with Status Code 400. I've used @SerializedName("postal_code")(made the change for all fields) and the test is now passing. Maybe there are some issue with the integration of JsonProperty? 

Also, I've updated jackson-annotations from pom.xml to v2.9.9 but the problem still persists.

File: src/test/java/api/cart/PostCartTests.java (hunk 2 above)
Comment (vladbm-work) [reply src/test/java/api/cart/PostCartTests.java]: I've modified the data for the test.

**Review by softvision-adrian-rosu — APPROVED** [review]
looks good to me

**Pull request merged**